### PR TITLE
Split changelog for each client platform

### PR DESCRIPTION
**Pull request by lindydonna**

### CODE REVIEW

File: CHANGELOG.android.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Azure Mobile Services Android SDK Change Log
+
+### Android SDK: Version 1.1.5
```
Comment (phvannor) [inline CHANGELOG.android.md]: nit: you might as well remove the platform from the headers

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,230 +1,9 @@
 # Azure Mobile Services Change Log
 
-### iOS SDK: Version 2.0
-- GA of offline sync changes from previous betas
+## [iOS SDK](CHANGELOG.ios.md)
```
Comment (phvannor) [inline CHANGELOG.md]: We should put these links on the readme.md in the changelog section and have it no longer point to this changelog.

File: CHANGELOG.android.md (hunk 1 above)
Comment (lindydonna) [inline CHANGELOG.android.md]: Yeah, I thought of that, but it doesn't hurt to be more specific.

File: CHANGELOG.md (hunk 2 above)
Comment (lindydonna) [inline CHANGELOG.md]: Will do.

**Pull request merged**